src/components/PDFView/index.js

```diff
@@ -49,42 +73,59 @@ class PDFView extends PureComponent {
             ? pdfContainerWidth : variables.pdfPageMaxWidth;
         const pageWidth = this.props.isSmallScreenWidth ? this.state.windowWidth - 30 : pageWidthOnLargeScreen;
 
+        const pdfStyle = [styles.PDFView];
+        const containerStyle = [styles.PDFView, this.props.style];
+
+        // If we're requesting a password then we need to set the background to
+        // defaultModalContainer color (white) and hide - but still render -
+        // the PDF component.
+        if (this.state.shouldRequestPassword) {
+            pdfStyle.push(styles.invisible);
+            containerStyle.push(styles.defaultModalContainer);
+        }
+
         return (
+
             <View
-                style={[styles.PDFView, this.props.style]}
+                style={containerStyle}
                 onLayout={event => this.setState({windowWidth: event.nativeEvent.layout.width})}
             >
-                <Document
-                    loading={<FullScreenLoadingIndicator />}
-                    file={this.props.sourceURL}
-                    options={{
-                        cMapUrl: 'cmaps/',
-                        cMapPacked: true,
+                <View style={pdfStyle}>
+                    <Document
+                        loading={<FullScreenLoadingIndicator />}
+                        file={this.props.sourceURL}
+                        options={{
+                            cMapUrl: 'cmaps/',
+                            cMapPacked: true,
+                        }}
+                        externalLinkTarget="_blank"
+                        onLoadSuccess={this.onDocumentLoadSuccess}
+                        onPassword={this.initiatePasswordChallenge}
+                    >
+                        {_.map(_.range(this.state.numPages), (v, index) => (
+                            <Page
+                                width={pageWidth}
+                                key={`page_${index + 1}`}
+                                pageNumber={index + 1}
+                            />
+                        ))}
+
+                    </Document>
+                </View>
+
+                {this.state.shouldRequestPassword && (
+                    <PDFPasswordForm
+                        onSubmit={password => this.onPasswordCallback(password)}
+                        isPasswordInvalid={this.state.isPasswordInvalid}
+                    />
+                )}
```

Comment:
Same here remove unnecessary line breaks in between component usage in render method